apps/admin-gui/src/app/shared/components/dialogs/create-sponsored-member-dialog/create-sponsored-member-dialog.component.ts

```diff
@@ -54,27 +59,56 @@ export class CreateSponsoredMemberDialogComponent implements OnInit {
   constructor(private dialogRef: MatDialogRef<CreateSponsoredMemberDialogComponent>,
               @Inject(MAT_DIALOG_DATA) private data: CreateSponsoredMemberDialogData,
               private membersService: MembersManagerService,
-              private store: StoreService) {
+              private store: StoreService,
+              private translator: TranslateService) {
   }
 
   ngOnInit(): void {
     this.loading = true;
     this.theme = this.data.theme;
-    this.parseNamespace();
-    if (this.namespaceOptions.length === 0) {
-      this.functionalityNotSupported = true;
+
+    this.membersService.getAllNamespacesRules().subscribe(rules => {
+      if (this.store.get('allow_empty_sponsor_namespace')) {
```

Comment:
I would add this property to the default config with a value `false` so it is clear, that such value exists.